Icon/Icon.module.less

```diff
@@ -22,7 +22,7 @@
 		text-indent: @agate-icon-text-indent;
 
 		&.huge {
-			background-position: center -((@agate-icon-sprite-huge-size - @agate-icon-huge-size) / 2);
```

Comment:
We'll have to remember this was here, in case we get complaints that the other themes' icons don't work properly.

Reply:
@Djspaceg Should this change be applied to the other sizes too?

Reply:
Oh! Yes. It should. I didn't notice where this was nested.

Reply:
Changing this to `center center` in the other sizes break the png and svg samples in the sampler.

Reply:
On closer inspection, the assets in sampler are sprites with sizes that won't center the icon with just `center`. It seems like we need to allow custom positioning for non-font icons.

Reply:
On closer inspection, the assets in sampler are sprites with sizes that won't center the icon with just center. It seems like we need to allow custom positioning for non-font icons.

Reply:
We could just fix these assets too. It may also be appropriate to have a more holistic approach to this images/sprites support case. One approach could be that we allow a consumer of Icon/IconButton to indicate how many sprites are in their image asset, defaulting to just `1`. This value can be relayed to the LESS via a custom CSS property like `--agate-icon-sprite-count` and we use that to properly calculate the vertical offset of the image in the `background-position` rule. WDYT? Shouldn't actually be difficult, and will work across skins, so images still look correct when a skin is changed.

Reply:
@teckliew, I could make this change if you'd prefer.

Reply:
Oh yea go ahead, if you're up for it 👍

Reply:
All done! It's now also based on #147 to save some work later.

Docs added, all sizes tested with regular icons, images, sprite images and simplified the legacy junk we no longer need.